blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeUtil.java

```diff
@@ -47,8 +47,20 @@
         } else if (execution.getCauseOfFailure() != null) {
             result = getStatus(execution.getCauseOfFailure());
             state = BlueRun.BlueRunState.FINISHED;
-        } else if (execution.isComplete()) {
-            result = BlueRun.BlueRunResult.SUCCESS;
+        } else if (execution.isComplete() &&  run.getResult() != null) { //workaround for https://issues.jenkins-ci.org/browse/JENKINS-38049
+            switch (run.getResult().toString()){
+                case "SUCCESS":
+                    result = BlueRun.BlueRunResult.SUCCESS;
+                    break;
+                case "ABORTED":
+                    result = BlueRun.BlueRunResult.ABORTED;
+                    break;
+                case "FAILURE":
+                    result = BlueRun.BlueRunResult.FAILURE;
+                    break;
+                default:
+                    result = BlueRun.BlueRunResult.UNKNOWN;
```

Comment:
Seems like the above `switch` could be shortened to ... 

```java
try {
    result = BlueRun.BlueRunResult.valueOf(run.getResult().toString());
} catch (RuntimeException e) {
    result = BlueRun.BlueRunResult.UNKNOWN;
}
```

That also covers states that might be added in the future.